kong/dao/migrations/cassandra.lua

```diff
@@ -423,4 +423,27 @@ return {
     end,
     down = function(_, _, dao) end
   },
+  {
+    name = "2017-07-31-113200_jwt_preflight_default",
+    up = function(_, _, dao)
+      local rows, err = dao.db:query([[
+        SELECT * FROM plugins WHERE name = 'jwt';
```

Comment:
This is how plugins migrations are already handled. The configuration of the plugins themselves does not belong to the core, but to the plugins themselves (and rightfully so).